pages/college_predictor.js

```diff
@@ -4,154 +4,157 @@ import styles from "./college_predictor.module.css";
-                                            index % 2 === 0
-                                                ? styles.even_row
-                                                : styles.odd_row
-                                        }
-                                    >
-                                        <td className={styles.cell}>
-                                            {item["College Rank"]}
-                                        </td>
-                                        <td className={styles.cell}>
-                                            {item["State"]}
-                                        </td>
-                                        <td className={styles.cell}>
-                                            {item.Institute}
-                                        </td>
-                                        <td className={styles.cell}>
-                                            {item["Academic Program Name"]}
-                                        </td>
-                                        <td className={styles.cell}>
-                                            {item["Opening Rank"]}
-                                        </td>
-                                        <td className={styles.cell}>
-                                            {item["Closing Rank"]}
-                                        </td>
-                                        <td className={styles.cell}>
-                                            {item["Quota"]}
-                                        </td>
-                                    </tr>
-                                ))}
-                            </tbody>
-                        </table>
-                    </div>
-                )}
+  return (
+    <div className={styles.container}>
+      <div className={styles.content}>
+        <h1>{getConstants().TITLE}</h1>
+        <h2>
+          {exam != "NEET"
+            ? "Your Category Rank: " + rank
+            : "Your Rank: " + rank}
+        </h2>
+        <h3>Chosen Round Number: {roundNumber}</h3>
+        <h3>Chosen Exam: {exam}</h3>
+        {exam != "NEET" && (
+          <>
+            <h3>Chosen Gender: {gender}</h3>
+            <h3>Chosen Home State: {stateName}</h3>
+          </>
+        )}
+        <h3>Predicted colleges and courses for you</h3>
+        {isLoading ? (
+          <div className={styles.loading}>
+            <div className={styles.spinner}></div>
+            <p>Loading...</p>
+          </div>
+        ) : (
+          <div>
+            <div>
+              <p className={styles.paragraph}>AI: All India</p>
+              <p className={styles.paragraph}>HS: Home State </p>
+              <p className={styles.paragraph}>OS: Out of State</p>
             </div>
-        </div>
-    );
+            <table className={styles.table}>
+              <thead>
+                <tr className={styles.header_row}>
+                  <th>Institute Rank</th>
```

Comment:
why not have a table component here too? isn't it almost like ScholarshipTable ? except there is no expansion here?